### ISSUE TITLE
Fixing config naming across roles

### ISSUE THREAD

**Issue by jbesraa** [stratum-mining/stratum#1260]
Config object across different roles is not well named. For example JDC config is called `ProxyConfig` we should change it to `JDCConfig` and so on for all other roles `[RoleName]Config`.

**Comment by devworlds** [comment]
Basically, need to change the ProxyConfig Object to JDCConfig, and update all references that use it?
![image](https://github.com/user-attachments/assets/687545ac-4223-4590-85c7-752b57d69899)

**Comment by jbesraa** [comment]
Updated top comment - We only have translator and mining proxy left.

**Comment by Sansh2356** [comment]
@plebhash  @jbesraa is this still pending ?

**Comment by jbesraa** [comment]
Yes, for the translator and mining-proxy. See top comment for more info or refer to #1493 and #1507
